clio-util/src/main/scala/org/broadinstitute/clio/util/auth/AuthUtil.scala

```diff
@@ -21,17 +22,25 @@ object AuthUtil extends ModelAutoDerivation with LazyLogging {
 
   private val shellOutCreds: Either[Throwable, OAuth2Credentials] = {
     logger.debug("Shelling out to gcloud to get credentials")
-    Right(ExternalGCloudCredentials)
+    // Need to test out the gcloud command here before we build the credentials
+    // Since the access token is lazily evaluated.
+    Either
+      .catchNonFatal(Process("gcloud", List("auth", "print-access-token")).!!.trim)
```

Comment:
Instead of duplicating this (which already exists  in `ExternalGCloudCredentials`), why not:
```scala
catchNonFatal {
  val _ = ExternalGCloudCredentials.refreshAccessToken()
  ExternalGCloudCredentials
}
```